projects/packages/stats-admin/src/class-wp-dashboard-odyssey-widget.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * A class that adds Jetpack Stats widget for WP Dashboard.
+ *
+ * @package automattic/jetpack-stats-admin
+ */
+
+namespace Automattic\Jetpack\Stats_Admin;
+
+/**
+ * Responsible for loading Jetpack Stats widget for WP Dashboard.
+ *
+ * @package jetpack-stats-admin
+ */
+class WP_Dashboard_Odyssey_Widget {
+
+	/**
+	 * Renders the widget and fires a dashboard widget action.
+	 */
+	public static function render() {
+		?>
+		<div id="dashboard_stats" class="jp-stats-widget" style="min-height: 200px;">
+			<div class="hide-if-js"><?php esc_html_e( 'Your Jetpack Stats widget requires JavaScript to function properly.', 'jetpack-stats-admin' ); ?></div>
+			<div class="hide-if-no-js" style="height: 100%">
+				<img
+					class="jp-stats-widget-loading-spinner"
+					width="32"
+					height="32"
+					style="position: absolute; left: calc(50% - 28px); top: calc(50% - 36px);"
+					alt=<?php echo esc_attr( __( 'Loading', 'jetpack-stats-admin' ) ); ?>
+					src="//en.wordpress.com/i/loading/loading-64.gif"
+				/>
+			</div>
+		</div>
+		<?php
+		/**
+		 * Fires when the dashboard is loaded, but no longer used anywhere in the Jetpack plugin.
+		 * The action is still available for backward compatibility.
+		 *
+		 * @since 3.4.0
+		 */
+		do_action( 'jetpack_dashboard_widget' );
+	}
+
+	/**
+	 * Load the admin scripts.
+	 */
+	public function load_admin_scripts() {
+		$config_data = ( new Odyssey_Config_Data() )->get_data();
+		// The widget doesn't use redux.
+		unset( $config_data['intial_state'] );
+		// TODO: change `configData` to a more unique name to avoid future conflicts.
+		( new Odyssey_Assets() )->load_admin_scripts( 'jetpack_stats_widget', 'widget-loader.min', 'configData', $config_data );
```

Comment:
Could we do this in this PR?

Reply:
The Odyssey side would a bit of work as well and that's the reason I pushed it back - added an issue [here](https://github.com/Automattic/wp-calypso/issues/75257) to track it. Will definitely finish the change before it going live 😄

Reply:
Sounds good! 👍